server/src/main/java/org/elasticsearch/action/ingest/SimulatePipelineRequest.java

```diff
@@ -160,7 +159,6 @@ static Parsed parse(Map<String, Object> config, boolean verbose, IngestService i
         return new Parsed(pipeline, ingestDocumentList, verbose);
     }
 
-    @UpdateForV10(owner = UpdateForV10.Owner.DATA_MANAGEMENT) // Unconditionally deprecate the _type field once V8 BWC support is removed
```

Comment:
I moved this to the `Metadata.TYPE` definition. It no longer applies here now that the `V_7` ref is removed. However I didn't want to remove the reminder.